.travis.yml

```diff
@@ -21,8 +21,6 @@ env:
     - INSTALL_PREFIX="$HOME/nvim-install"
     # Log directory for Clang sanitizers and Valgrind.
     - LOG_DIR="$BUILD_DIR/log"
-    # Nvim log file.
-    - NVIM_LOG_FILE="$BUILD_DIR/.nvimlog"
```

Comment:
@blueyed That's part of the temporary commits, I am puzzled about why travis is failing.